frontend/src/core/comments/components/CommentsList.js

```diff
@@ -44,7 +44,7 @@ export default function CommentsList(props: Props) {
                 user={ translation.user }
                 username={ translation.username }
                 imageURL={ translation.userGravatarUrlSmall}
-                translationId={ translation.pk }
+                id={ translation.pk }
```

Comment:
I would avoid using `id` as a prop name, as it can be confused with the `id` attribute of DOM nodes. I thought `translationId` was fine here, why change it?